server/routes/reporting/standard-report/definitions.js

```diff
@@ -203,6 +206,27 @@
         bespokeReportBody: true,
         exportable: true,
       },
+      'reasonable-adjustments': {
+        title: 'Reasonable adjustments report',
+        apiKey: 'ReasonableAdjustmentsReport',
+        search: 'dateRange',
```

Comment:
See previous - but also does it make sense for this to be `fixedDateRange` or similar and handle it that way?